index.html

```diff
@@ -1544,8 +1547,23 @@ <h2>Dependencies</h2>
     </p>
   </section>
 
+  <section class="informative">
+    <h2>Accessibility considerations</h2>
+    <p>
+      This document specifies a set of metadata, enabling developers and publishers to describe and configure the behavior and appearance of MiniApps. Some manifest attributes, like `color_scheme` and the `window` resource members, aim user agents to generate customized user interfaces to enable interaction between the user and the MiniApp.
+    </p>
+    <p>
+      The <a><code>icons</code></a> member allows developers to describe the visual and iconic representation of a MiniApp in the platform and operating system. Here, the platform should adequately communicate with the platform accessibility services to represent the images, also using the alternative textual descriptions (e.g., <a><code>icon</code></a>'s <a href="https://www.w3.org/TR/image-resource/#dfn-label" data-link-type="dfn"><code>label</code></a>, <a data-link-for="MiniApp manifest" data-link-type="dfn"><code>name</code></a> and <a><code>short_name</code></a>) to enhance accessibility (i.e., icons in the home screen of the device, listing in app directories, and app configuration).
+    </p>
+    <p>
+      User agents must ensure that the user interface is perceivable and operable, so they should take special consideration when processing the members that affect the user agent interface and the rendered content, like with the setup of color schemes and themes (i.e., <a><code>color_scheme</code></a>, <a data-link-for="MiniApp window resource" data-link-type="dfn"><code>background_color</code></a>, <a><code>background_text_style</code></a>, <a><code>navigation_bar_background_color</code></a>, <a><code>navigation_bar_background_color</code></a>, <a><code>navigation_bar_text_style</code></a>, and <a><code>navigation_style</code></a>) and other configuration settings like <a><code>orientation</code></a>, <a><code>fullscreen</code></a>, <a><code>enable_pull_down_refresh</code></a>, and <a><code>on_reach_bottom_distance</code></a>.
+    </p>
+    <p>
+      User agents interpreting this manifest should help users orient within and control windows and viewports. They should also provide alternative views addressing the different usage scenarios through the advanced use of the `device_type` member and extending the manifest with vendor-specific members to enhance accessibility. Thus, MiniApp user agents and platforms should provide mechanisms for configuring user stylesheets, themes and customize the display and interaction with the application, complying with the applicable specifications and conventions, in particular the [[[UAAG20]]].
+    </p>
+  </section>
   <section>
-    <h2>Security and Privacy Considerations</h2>
+    <h2>Security and privacy considerations</h2>
```

Comment:
This change is not related to accessibility considerations, and we need to change the section name per https://github.com/w3c/miniapp-manifest/issues/31

Reply:
Thanks @xfq. Reverting this non-related change.